packages/vue3-vite/src/generators/vue3-vite/files/tsconfig.json.template

```diff
@@ -8,6 +8,7 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "target": "ESNext",
+    "resolveJsonModule": true,
```

Comment:
I added this property to `apps/new-app/tsconfg.json.template` instead of the root `tsconfig.base.json`.
But I wonder if there would be any performance implications if I just add it to the root `tsconfig.base.json`?

Reply:
Good question! I think it's acceptable to put it in the root.